### PR TITLE
Fr us 48

### LINKED ISSUE #51: User Story 48, Merchant cannot edit an item if details are bad/missing
As a merchant employee
When I try to edit an existing item
If any of my data is incorrect or missing (except image)
Then I am returned to the form
I see one or more flash messages indicating each error I caused
All fields are re-populated with my previous data

### PR DESCRIPTION
**Pull request by fredrondina**
added test to validate merchants cant edit items to have attributes that dont make sense.

Added validation to item model that its inventory cant be negative 

Had to change one test where the syntax of the error messages changed after I added this new validation to the item model

closes #51

### CODE REVIEW

File: spec/features/merchant/items_spec.rb [hunk 1]
```diff
@@ -116,6 +116,28 @@
       expect(page).to have_content("Skinny Tires")
       expect(page).to have_content("Half the tire at twice the cost :^)")
     end
+
+    it "wont let me edit items to have attributes that arnt valid" do
```
Comment (kelshadarby) [inline spec/features/merchant/items_spec.rb]: Just a quick thing to change this to aren't

**Review by kelshadarby — APPROVED** [review]

File: app/models/item.rb [hunk 2]
```diff
@@ -11,6 +11,7 @@ class Item <ApplicationRecord
                         :inventory
   validates_inclusion_of :active?, :in => [true, false]
   validates_numericality_of :price, greater_than: 0
+  validates_numericality_of :inventory, greater_than: 0
```
Comment (brian-greeson) [inline app/models/item.rb]: Should this be greater than or equal to?

**Review by brian-greeson — CHANGES_REQUESTED** [review]
I think the validation needs tweaking. see comments

File: spec/features/merchant/items_spec.rb (hunk 1 above)
Comment (brian-greeson) [reply spec/features/merchant/items_spec.rb]: Florida

**Review by brian-greeson — APPROVED** [review]
Looks Great

**Pull request merged**